client/src/pages/TaskFormPage/TaskFormPage.js

```diff
@@ -85,54 +85,59 @@ const TaskFormPage = () => {
 
   return (
     <div className={styles.bg}>
-      <p className={styles.header}>Create a Task</p>
-      <div className={styles.inputBox}>
-        <InputBox
-          header={"Task Title"}
-          type={"text"}
-          placeholder={"Task Name"}
-          onChange={(e) => updateTaskFormData(e, "title")}
-          value={taskFormData.title}
-        />
-        <InputBox
-          header={"Location"}
-          type={"text"}
-          placeholder={"Location Name"}
-          onChange={(e) => updateTaskFormData(e, "location")}
-          value={taskFormData.location}
-        />
-        <InputBox
-          header={"Description"}
-          type={"text"}
-          placeholder={""}
-          expanded={true}
-          onChange={(e) => updateTaskFormData(e, "description")}
-          value={taskFormData.description}
-        />
-        <p className={styles.inputHeader}>Start</p>
-        <DateSelector
-          showTime={true}
-          selectedDate={taskFormData.startDate}
-          onDateChanged={(newDate) => updateTaskFormData(newDate, "startDate")}
-        />
-
-        <p className={styles.inputHeader}>End</p>
-        <DateSelector
-          showTime={true}
-          selectedDate={taskFormData.endDate}
-          onDateChanged={(newDate) => updateTaskFormData(newDate, "endDate")}
-        />
-      </div>
+      <div>
+        <p className={styles.header}>Create a Task</p>
+        <div className={styles.inputBox}>
+          <InputBox
+            header={"Task Title"}
+            type={"text"}
+            placeholder={"Task Name"}
+            onChange={(e) => updateTaskFormData(e, "title")}
+            value={taskFormData.title}
+          />
+          <InputBox
+            header={"Location"}
+            type={"text"}
+            placeholder={"Location Name"}
+            onChange={(e) => updateTaskFormData(e, "location")}
+            value={taskFormData.location}
+          />
+          <InputBox
+            header={"Description"}
+            type={"text"}
+            placeholder={""}
+            expanded={true}
+            onChange={(e) => updateTaskFormData(e, "description")}
+            multiline={true}
+            value={taskFormData.description}
+          />
+          <p className={styles.inputHeader}>Start</p>
+          <DateSelector
```

Comment:
For the date selector, is there any way to extend the box so it matches the length of the other boxes? (Same with the description box)
![image](https://user-images.githubusercontent.com/24628243/163659510-eed49423-07ec-4bc7-a219-e85e5d4831a8.png)

Reply:
Unless this was the intended design that we wanted

Reply:
Yep, this is the intended design. By default, the button actually expands bigger than this, but I've added a `max-width` of 120px to prevent that.

Reply:
Noted